tools/unit_tests/test_splitter.py

```diff
@@ -8,7 +8,7 @@
 
 
 def execute_test(obj):
-    for i in range(10):
+    for _ in range(10):
```

Comment:
Chris may have something to say about this. I haven't a big problem with it, but just a warning.